packages/app/src/vis-packs/core/complex/ComplexToolbar.tsx

```diff
@@ -83,6 +81,18 @@ function ComplexToolbar() {
 
       <Separator />
 
+      <Selector
+        value={visType}
+        onChange={(value: ComplexVisType) => setVisType(value)}
+        options={Object.values(ComplexVisType)}
+        optionComponent={({ option }) => (
+          // eslint-disable-next-line react/jsx-no-useless-fragment
+          <>{`${VIS_TYPE_SYMBOLS[option]} ${option}`}</>
+        )}
+      />
+
+      <Separator />
```

Comment:
Moved this down for consistency with the complex line toolbars (scale type then vis type).